nextflow_TES/README.md

```diff
@@ -10,7 +10,7 @@ Instructions below have been tested for CSC Rahti cloud running on RedHat OpenSh
         # Cloud parameters
         export CLOUD_BASE_URL=c03.k8s-popup.csc.fi:8443
         export CLOUD_NAMESPACE=tesk-cineca
-        export NEXTFLOW_POD=nextflow-dockerhub-7-9bdlm
+        export NEXTFLOW_POD=nextflow-dockerhub-10-gcdt8
```

Comment:
This pod name will keep changing whenever pod is rebuild by user or is involuntary build up by OpenShift cluster due to pod rescue, load balancing etc..Shall we mention it in readme so that reader doesn't get confused that it is related to hardcoded Nextflow container name/tags

Reply:
Good point. I reflected it here: https://github.com/CINECA-project/wp4-federated-joint-cohort-analysis/pull/20/commits/23691298a96d4b54bcae72555f8a81c83f74a16a